system/t02_config/ConfigShowTest_gold

```diff
@@ -15,6 +15,7 @@
     "gpgDisableVerify": false,
     "gpgProvider": "gpg",
     "downloadSourcePackages": false,
+    "packagePoolStorage": {},
```

Comment:
I think you need to add `"type": "local"` here or update the code to default to empty and add code to deal with empty config. In the form of current code, this will fail the test due to mismatching between `config.go` and content here. Same for `CreateConfigTest_gold` below.

Reply:
Ack, that's what I get for not re-running the full test suite when I changed the config. As-is it already handles this being blank, so I just updated it to not bother adding anything to it if everything is still in the default values.